Testscripts/Linux/XDPDumpSetup.sh

```diff
@@ -110,6 +112,25 @@ function Run_XDPDump {
     fi
 }
 
+# Check if kernel supports XDP or not
+function check_xdp_support {
```

Comment:
Can we define it as an utility function to be used by all other testscripts ?

Reply:
As I mentioned above, This is a setup script and it is used by all xdp testscript. I don't see any necessary of adding it to utility